doc/source/Reference/ScriptingReference/Metadata/index.md

```diff
@@ -0,0 +1,115 @@
+makes it easy to customise the UI for a specific node type, or even for a
+specific node instance. This document provides a reference for the most common
+metadata items.
+
+General
+-------
+
+```eval_rst
+=================== =============================== ============================
+Name                Purpose                         Example Values
+=================== =============================== ============================
+label               Label used instead of plug name "My Label"
+description			Describes the purpose of a      "Turns on the thingammajig"
+                    node or plug.
+userDefault         Overrides the default value     10.5
+preset:<name>       Specifies a named preset value  "preset:Max", 1
+=================== =============================== ============================
+```
+
+NodeEditor Layout
+-----------------
+
+```eval_rst
+=================== =============================== ============================
+Name                Purpose                         Example Values
+=================== =============================== ============================
+layout:divider		Places a divider after the plug True
+layout:index        Integer index in the layout	    0 (first), -1 (last)
+                    order
+layout:section      Specifies the section the plug  TabName.SectionName
+                    belongs in
+layout:accessory    Places widget on same line as   True
+                    previous widget
+=================== =============================== ============================
+```
+
+NodeGraph Layout
+----------------
+
+```eval_rst
+====================== ================================ ================================
+Name                   Purpose                          Example Values
+====================== ================================ ================================
+nodule:color           The colour of the plug           IECore.Color3f( 0, 1, 0 )
+connectionGadget:color The colour of input connections  IECore.Color3f( 1, 0, 0 )
+nodeGadget:color       The colour of a node             IECore.Color3f( 0, 0, 1 )
+noduleLayout:section   The edge the plug appears on     "left", "right", "top", "bottom"
+noduleLayout:visible   Shows/hides the plug             True (visible), False (hidden)
+====================== ================================ ================================
+```
+
+Viewer Layout
+-------------
+
+```eval_rst
+====================== ================================ ================================
+Name                   Purpose                          Example Values
+====================== ================================ ================================
+layout:divider		   Places a divider after the plug  True
+layout:index           Integer index in the layout	    0 (first), -1 (last)
+                       order
+toolbarLayout:section  The edge of the viewer the plug  "Left", "Right", "Top", "Bottom"
```

Comment:
Oof. I never realized we use "Left" here and "left" for "noduleLayout:section". Seems like we should make that consistent at some point.